erpnext/www/book-appointment/index.js

```diff
@@ -0,0 +1,213 @@
+
+frappe.ready(() => {
+    initialise_select_date()
+})
+window.holiday_list = [];
+
+async function initialise_select_date() {
+    navigate_to_page(1);
+    await get_global_variables();
+    setup_date_picker();
+    setup_timezone_selector();
+    hide_next_button();
+}
+
+async function get_global_variables() {
+    window.appointment_settings = (await frappe.call({
```

Comment:
Using .then() would be better than `await` at the moment, but it's fine for now. Just add a comment explaining await as an alternative to `.then` in one short line for people looking at the code in the future.

P.S. In both frappe and ERPNext we have stuck to using `.then()` everywhere.